rbx_dom_weak/src/brick_color.rs

```diff
@@ -344,6 +367,13 @@ make_brick_color!({
 });
 
 impl BrickColor {
+    /// Convert from a BrickColor palette number to a BrickColor.
+    ///
+    /// This function should not be used for new code, it only exists because of
+    /// a misunderstanding of the range of valid BrickColor values.
+    ///
+    /// This function operates like `BrickColor.palette` within Roblox.
+    #[deprecated]

```

Comment:
```suggestion
    #[deprecated(
        since = "VERSION",
        note = "Please use BrickColor::from_number instead."
    )]
```

Reply:
My note doesn't mention that they aren't equivalent, but documentation should.

Reply:
I guess the next release should be 1.9.0